src/main/java/spiffe/provider/SpiffeSVID.java

```diff
@@ -0,0 +1,69 @@
+package spiffe.provider;
+
+import spiffe.api.svid.Workload;
+
+import java.security.PrivateKey;
+import java.security.cert.X509Certificate;
+import java.util.Set;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Represents a SPIFFE Identity
+ *
+ */
+class SpiffeSVID {
```

Comment:
Will this naming still work when we update this code to be compatible with JWT-SVID as well? I'm not sure if this will be applicable at that time or not, but in general we probably want to be explicit about SVID type whenever mentioning it

Reply:
would be _SpiffeX509SVID_, right ?